src/Analyzers/CSharp/Tests/MakeLocalFunctionStatic/MakeLocalFunctionStaticTests.cs

```diff
@@ -394,6 +395,71 @@ static int fibonacci(int n)
         }}
     }}
 }}",
+parseOptions: CSharp8ParseOptions);
+        }
+
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsMakeLocalFunctionStatic)]
+        [WorkItem(46858, "https://github.com/dotnet/roslyn/issues/46858")]
+        public async Task TestMissingIfAnotherLocalFunctionCalled()
+        {
+            await TestMissingAsync(
+@"using System;
+
+class C
+{
+    void M()
+    {
+        void [||]A()
+        {
+            B();
+        }
+
+        void B()
+        {
+        }
+    }
+}", parameters: new TestParameters(parseOptions: CSharp8ParseOptions));
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsMakeLocalFunctionStatic)]
+        public async Task TestCallingOtherLocalFunction()
```

Comment:
This test just validates that existing behaviour wasn't broken so I didn't add the `WorkItem` attribute. Is that correct?

Also didn't add a test for the recursive case, as its already covered by another test in this class.